vassal-app/src/main/java/VASSAL/configure/StringEnumConfigurer.java

```diff
@@ -49,6 +52,14 @@ public Component getControls() {
       panel = new ConfigurerPanel(getName(), "[]", "[]rel[]"); // NON-NLS
 
       box = new JComboBox<>(validValues);
+      // Ensure the text stays visible when the combobox is disabled
+      box.setRenderer(new DefaultListCellRenderer() {
+        @Override
+        public void paint(Graphics g) {
+          setForeground(Color.BLACK);
```

Comment:
Should this be set to whatever the "enabled" color is? It won't necessarily be black in all cases.

Reply:
Good catch. Improved version committed that respects the existing UI colors.